better-i18n-for-your-angular-apps/proposal.yaml

```diff
@@ -0,0 +1,35 @@
+---
+Session:
+  Title: Better i18n for your Angular apps
+  Description: >
+    Internationalization and Localization are the two fundamental procedures
+    to make applications available in different languages and cultures. However,
+    when taking a closer look, there’s much more coming into play when
+    internationalizing an application. Starting from translators and their tools,
+    over a properly integrated development workflow, to actual templates that
+    consume translations. Since Angular comes with rather poor support for i18n,
+    the community has built their own extensions. This should no longer be. In
+    this talk, Chirayu and Pascal are going to show you their plans to bring
+    first-class i18n support to the Angular framework.
+  Length: 20 # sessions are 5 or 20 minutes long
+  Tags:
+    - i18n
+
+Speaker:
+  name: Chirayu Krishnappa
+  twitter: 
+  email: chirayu@google.com
+  bio: >
+    Software Engineer on the core Angular team at Google.
```

Comment:
Chirayu is a core Angular team member and has contributed to AngularJS,
    AngularDart and Protractor. He loves technology. When not working, he's
    learning music, improv or painting.